### PR TITLE
Adds throttleInterval control

### PR DESCRIPTION
**Pull request by markgoodyear**
A minor PR to allow control the throttle interval, this could be handy depending on the use case of Hunt and may provide performance benefits.

### CODE REVIEW

**Comment by jeremenichelli** [comment]
Interesting! I think that I would add a short explanation on what the developer should expect if it increases the value and what the default is.

Do you think you can do that Mark? If not I can come up with something over the week.

**Comment by markgoodyear** [comment]
@jeremenichelli Sure, I'll update the readme today 👍

File: dist/hunt.js [hunk 1]
```diff
@@ -111,6 +112,9 @@
   // fallback leave function to avoid sanity check
   Hunted.prototype.leave = noop;
 
+  // throttle interval
+  Hunted.prototype.throttleInterval = DEFAULT_THROTTLE_INTERVAL;
```
Comment (jeremenichelli) [inline dist/hunt.js]: I think this line is not necessary since we are actually not using this value when initializing the event listeners.

Comment (markgoodyear) [reply dist/hunt.js]: @jeremenichelli You're right, I've pushed an update to my fork, it should come through here soon 👍

**Pull request merged**